### DISCUSSION TITLE
请问未来有没有计划移植到VR版本？

### DISCUSSION THREAD

**Discussion by ShuoShen** [jynew/jynew#1511] [Ideas]
想象可以把这个游戏移植到VR。 
第一种方案是做第三方视角，类似Moss。 
第二种方案是第一视角， 沉浸式体验。 

第一种看上去可行性更高。 

请问各位contributor 有没有想过VR移植计划。 实现VR移植的最大障碍在哪？

**Comment by MephistoSatanDevil** [comment]
穷，没有vr设备以供开发
